src/Cli/Microsoft.DotNet.Cli.Utils/Uuid.cs

```diff
@@ -27,7 +27,7 @@ public static Guid Create(string name)
             Array.Copy(namespaceBytes, streamToHash, namespaceBytes.Length);
             Array.Copy(nameBytes, 0, streamToHash, namespaceBytes.Length, nameBytes.Length);
 
-            var hashResult = XxHash3.Hash(streamToHash); // This is just used for generating a named pipe so we don't need a cryptographic hash
```

Comment:
nit: `Debug.Assert(hashResult.Length >= 16);`

Reply:
I don't dislike this, but I'd rather keep this small. Maybe add this to the main-targeted PR?